src/service/modem_response.proto

```diff
@@ -5,11 +5,12 @@ import "service/attach_response.proto";
 import "service/scan_response.proto";
 import "service/gps_response.proto";
 
-message modem_response_t {
+message modem_response {
   oneof message {
     gps_response gps_response = 1;
     scan_response scan_response = 2;
     attach_response attach_response = 3;
   }
   repeated bytes signature = 4;
+  repeated bytes signer = 5;
```

Comment:
uhh.. thus us called `pub_key` everywhere else and would be needed to verify the signature anyway

Reply:
```
message blockchain_signature_v1 {
  bytes signer = 1;
  bytes signature = 2;
}
```

but if we prefer the pub_key / signature convention, that's fine.

Reply:
yeah just going by what the newer protos for all the grpc services do there. 

Thanks for adjusting